### PR TITLE
chore: bump core and rm ssz feat

### PR DESCRIPTION
**Pull request by yash-atreya**
## Motivation

Ref #1166 

## Solution

Bump core to `0.8.0`, rm `ssz` feature from meta-crate.

## PR Checklist

- [ ] Added Tests
- [ ] Added Documentation
- [ ] Breaking changes

### CODE REVIEW

File: crates/alloy/Cargo.toml [hunk 1]
```diff
@@ -259,7 +259,6 @@ getrandom = ["alloy-core/getrandom"]
 rand = ["alloy-core/rand"]
 rlp = ["alloy-core/rlp"]
 serde = ["alloy-core/serde", "alloy-eips?/serde", "dep:alloy-serde"]
-ssz = ["alloy-core/ssz", "alloy-rpc-types?/ssz"]
```
Comment (mattsse) [inline crates/alloy/Cargo.toml:262]: this we should keep but it would only pass through the alloy-rpc-types feature

**Review by mattsse — CHANGES_REQUESTED** [review]
smol nit

File: Cargo.toml [hunk 2]
```diff
@@ -68,19 +68,19 @@ alloy-transport-http = { version = "0.2", path = "crates/transport-http", defaul
...
 
 alloy-rlp = { version = "0.3", default-features = false }
 
 alloy-chains = { version = "0.1.18", default-features = false }
 
 # ethereum
-ethereum_ssz_derive = "0.5"
-ethereum_ssz = "0.5"
+ethereum_ssz_derive = "0.7"
+ethereum_ssz = "0.7"
```
Comment (DaniPopes) [inline Cargo.toml:83]: blocked on ssz 0.8

Comment (mattsse) [reply Cargo.toml:83]: why?

Comment (DaniPopes) [reply Cargo.toml:83]: nvm, missing impls tho

Comment (mattsse) [reply Cargo.toml:83]: sigh

Comment (DaniPopes) [reply Cargo.toml:83]: can workaround by just using Bloom::new() since it's implemented for FixedBytes<N> but not Bloom(FixedBytes)

Comment (yash-atreya) [reply Cargo.toml:83]: > sigh
Need `Bloom` impl

Comment (mattsse) [reply Cargo.toml:83]: @yash-atreya mind porting the previous alloy-core impl to ethereum-ssz ?

Comment (DaniPopes) [reply Cargo.toml:83]: not a blocker tho

**Review by mattsse — APPROVED** [review]

**Comment by mattsse** [comment]
I want to block this until we have a date for new release

**Pull request merged**